src/System.Management.Automation/engine/remoting/client/RemotingErrorRecord.cs

```diff
@@ -55,38 +54,6 @@ public RemotingErrorRecord(ErrorRecord errorRecord, OriginInfo originInfo)
             _originInfo = originInfo;
         }
 
-        #region ISerializable implementation
-
-        /// <summary>
-        /// Serializer method for class.
-        /// </summary>
-        /// <param name="info">Serializer information.</param>
-        /// <param name="context">Streaming context.</param>
-        public override void GetObjectData(SerializationInfo info, StreamingContext context)
-        {
-            if (info == null)
-            {
-                throw PSTraceSource.NewArgumentNullException(nameof(info));
-            }
-
-            base.GetObjectData(info, context);
-
-            info.AddValue("RemoteErrorRecord_OriginInfo", _originInfo);
-        }
-
-        /// <summary>
-        /// Deserializer constructor.
-        /// </summary>
-        /// <param name="info">Serializer information.</param>
-        /// <param name="context">Streaming context.</param>
-        protected RemotingErrorRecord(SerializationInfo info, StreamingContext context)
```

Comment:
protected constructor should be preserved.

Reply:
brought back.